components/table/test/table.sticky.visual-diff.html

```diff
@@ -91,7 +91,6 @@
 				<table class="d2l-table">
 					<thead>
 						<tr>
-							<th rowspan="2" style="width: 1%" class="top"><input type="checkbox"></th>
```

Comment:
Taking selection out of the "multi-row" header tests and creating 3 separate tests for selection below.